doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md

```diff
@@ -114,9 +114,10 @@ $ ./notification.sh http://172.17.0.13:5050/notify
 > Host: 172.17.0.13:5050
 > Content-Type: application/json
 > Accept: application/json
-> User-Agent: orion/0.10.0
+> User-Agent: orion/2.2.0
```

Comment:
Orion 0.10.0 is an ancient version. I have upgraded all the references to User-Agent to 2.2.0, which is newer.

NTC (informative)